DSharpPlus/Entities/Interaction/DiscordInteraction.cs

```diff
@@ -114,6 +114,16 @@ public async Task<DiscordMessage> EditOriginalResponseAsync(DiscordWebhookBuilde
             return await this.Discord.ApiClient.EditOriginalInteractionResponseAsync(this.Discord.CurrentApplication.Id, this.Token, builder);
         }
 
+        /// <summary>
+        /// Gets the original interaction response.
+        /// </summary>
+        /// <returns></returns>
+        /// <exception cref="Exceptions.NotFoundException">Thrown when the response does not exist.</exception>
+        /// <exception cref="Exceptions.BadRequestException">Thrown when an invalid parameter was provided.</exception>
+        /// <exception cref="Exceptions.ServerErrorException">Thrown when Discord is unable to process the request.</exception>
+        public async Task<DiscordMessage> GetOriginalResponseAsync(ulong messageId)
+            => await this.Discord.ApiClient.GetOriginalInteractionResponseAsync(this.Id, this.Token);
```

Comment:
This should probably take a concrete DiscordMessage, not an ID.

Reply:
`messageId` isn't used at all. I'll leave this to @Lulalaby to work out.

Reply:
oh i fucked up, sry

Reply:
Updated